src/stylesheets/components/_top-fold.scss

```diff
@@ -42,12 +42,17 @@ main {
   transform: rotate(40deg);
   width: 5em;
 
-  @media ($screen-xs-up) {
-    left: 25px;
+  @media ($screen-sm-up) {
+    left: 200px;
+    transform: scale(2.5) translateX(-300%) rotate(1600deg);
   }
+}
 
-  @media ($screen-sm-up) {
-    left: 170px;
+.yellow-pill-roll {
+
+  @media($screen-sm-up) {
+    transform: scale(1) translateX(0) rotate(40deg);
+    transition: transform 2s ease-out;
```

Comment:
im not really sure what your end goal is here, but this will only animate once. if these are supposed to animate infinitely, i recommend doing keyframe animations. if you need help with that, let me know

Reply:
ah ha, after seeing the gif, this makes sense